src/frontend/apps/desk/src/api/fetchWrapper.ts

```diff
@@ -0,0 +1,42 @@
+import useAuthStore from '@/auth/useAuthStore';
+
+interface FetchWrapperOptions {
+  withoutProxy?: boolean;
+}
+
+export const fetchWrapper = async (
+  input: RequestInfo,
+  init?: RequestInit,
+  options?: FetchWrapperOptions,
+): Promise<Response> => {
+  /**
+   * API Helper
+   */
+  if (
+    !options?.withoutProxy &&
+    typeof input === 'string' &&
+    input.startsWith('/api')
```

Comment:
More than happy if you can describe whenever we would fetch data from another source than the API

Reply:
I don't know what to say, a front can interact with something else than the backend. 
If I want to use a webservice from another domain that have the keyword `/api/`, it will replace the url, it will not work.